test/toHaveStyleRule.spec.js

```diff
@@ -137,16 +138,29 @@ test('complex string', () => {
   toHaveStyleRule(<Wrapper />, 'border', '1px solid rgba(0,0,0,0.125)')
 })
 
-test('undefined', () => {
+test('conditional rule - null value', () => {
   const Button = styled.button`
     cursor: ${({ disabled }) => !disabled && 'pointer'};
     opacity: ${({ disabled }) => disabled && '.65'};
   `
 
-  toHaveStyleRule(<Button />, 'opacity', undefined)
+  toHaveStyleRule(<Button />, 'opacity', null)
   toHaveStyleRule(<Button />, 'cursor', 'pointer')
   toHaveStyleRule(<Button disabled />, 'opacity', '.65')
-  toHaveStyleRule(<Button disabled />, 'cursor', undefined)
+  toHaveStyleRule(<Button disabled />, 'cursor', null)
+})
+
+test('at rules initially no value', () => {
+  const Button = styled.button`
+    @media (max-width: 640px) {
+      width: 50%;
+    }
+  `
+  notToHaveStyleRule(<Button />, 'width')
```

Comment:
@MicheleBertoli I mentioned those changes in our previous discussion, as promised I deliver those as a separate PR. Changes were made due to the 'notToHaveStyleRule' check marked with this comment. It is a quick fix and probably there is a better way to do it. If you share any ideas I can try to fix in better way. In my old test suits such tests worked well and a scenario when you check lack of some rule seems to be useful when doing mobile first design. Anyway, I would like to get your opinion on it.